### ISSUE TITLE
[Bug]: The app is reloading on changing pages.

### ISSUE THREAD

**Issue by kailashchoudhary11** [JasonDsouza212/free-hit#324]
### What happened?

When we try to navigate through the navbar, the app is reloading.
I would like to fix this bug.

### Add screenshots

Add screenshots

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by JasonDsouza212** [comment]
There are two navbars: one for the About page and the other for the rest of the pages. I think that's why you are experiencing the page load issue. If what I mentioned is not the problem causing it,  let me know so that I can assign this issue to you.

**Comment by kailashchoudhary11** [comment]
> There are two navbars: one for the About page and the other for the rest of the pages. I think that's why you are experiencing the page load issue. If what I mentioned is not the problem causing it, let me know so that I can assign this issue to you.

Not it is not because of what you mentioned. The pages are reloading because we're using anchor tags to navigate in the navbar. One of the main purposes of React is to avoid refreshing the site. We can use the `NavLink` component to achieve this.

**Comment by JasonDsouza212** [comment]
Do you want to work on this @kailashchoudhary11

**Comment by kailashchoudhary11** [comment]
Yeah, I will fix #339 first and then this. You can assign it to me.

**Comment by JasonDsouza212** [comment]
I have assigned this issue to you
